src/components/socialLogin/GoogleConfig.js

```diff
@@ -0,0 +1,3 @@
+export const ClientId = "1011517362493-p5a254c3evq53ossa7qr1hdollkfe568.apps.googleusercontent.com";
```

Comment:
Use a .env file to manage your environment variables

Reply:
@adesege worked on feedback